build.sbt

```diff
@@ -3,7 +3,9 @@ organization := "com.scaltrack"
 
 version := "1.0-SNAPSHOT"
 
-lazy val root = (project in file(".")).enablePlugins(PlayScala)
+lazy val root = (project in file("."))
+.enablePlugins(PlayScala, PlayNettyServer)
+  .disablePlugins(PlayAkkaHttpServer)
```

Comment:
Do you think that Netty is more performant than Akka Http?

Reply:
Try it and find out.  Historically the difference has been very small, but different benchmarks can work better for different engines.